db/migrate/20231023103344_add_subdomains_to_companies.rb

```diff
@@ -0,0 +1,8 @@
+class AddSubdomainsToCompanies < ActiveRecord::Migration[7.0]
+  def change
+    Organization.all.each do |organization|
+      organization.update(subdomain: organization.name.downcase.gsub(/\s+/, '-')) if organization.subdomain.nil?
```

Comment:
right now this is conflicting with the validation of the subdomain in organization model
`validates :subdomain, presence: true, uniqueness: true, format: { with: /\A\w+\z/ }`

So either the gsub should be underscore ('_') instead or update the validations to accept dashes ('-') too

Reply:
That's a good point, we'll update the validations